### PR TITLE
React-Query. Add safe `QueryKey` factories

### PR DESCRIPTION
**Pull request by stefanthaler**
cc: @turansky

### CODE REVIEW

File: kotlin-react-query/src/main/kotlin/react/query/QueryKey.ext.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@file:Suppress(
+    "NOTHING_TO_INLINE"
```
Comment (turansky) [inline kotlin-react-query/src/main/kotlin/react/query/QueryKey.ext.kt:2]: Missed trailing comma

File: kotlin-react-query/src/main/kotlin/react/query/QueryKey.ext.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+)
+
+import react.query.QueryKey
```
Comment (turansky) [inline kotlin-react-query/src/main/kotlin/react/query/QueryKey.ext.kt:5]: Redundant import?

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-react-query/src/main/kotlin/react/query/QueryKey.ext.kt (hunk 1 above)
Comment (stefanthaler) [reply kotlin-react-query/src/main/kotlin/react/query/QueryKey.ext.kt]: Fixed.

File: kotlin-react-query/src/main/kotlin/react/query/QueryKey.ext.kt (hunk 2 above)
Comment (stefanthaler) [reply kotlin-react-query/src/main/kotlin/react/query/QueryKey.ext.kt:5]: No, seems to be necessary.

Comment (turansky) [reply kotlin-react-query/src/main/kotlin/react/query/QueryKey.ext.kt:5]: Because `package` missed? :)

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-react-query/src/main/kotlin/react/query/QueryKey.ext.kt (hunk 2 above)
Comment (stefanthaler) [reply kotlin-react-query/src/main/kotlin/react/query/QueryKey.ext.kt]: Fixed.

**Review by turansky — APPROVED** [review]

**Pull request merged**